src/libecalc/fixtures/case_types.py

```diff
@@ -4,12 +4,12 @@
 from typing import Dict, NamedTuple, TextIO
 
 from libecalc import dto
-from libecalc.presentation.yaml.yaml_entities import Resource
+from libecalc.presentation.yaml.yaml_entities import MemoryResource
 
 
 @dataclass
 class YamlCase:
-    resources: Dict[str, Resource]
+    resources: Dict[str, MemoryResource]
```

Comment:
I haven't looked at the YamlCase, this change is because of the rename. I'm assuming we could manage with Resource, but somewhere in the tests headers and data properties are probably being used.